### PR TITLE
Revert "Splits preview link from open in column links"

### PR DESCRIPTION
**Pull request by SHession**
Reverts guardian/workflow-frontend#212. This PR has caused the preview option to no longer appear to most users, reverting for now.

### CODE REVIEW

**Comment by akash1810** [comment]
Is it worth making the preview link column one of the core columns that cannot be hidden?

**Comment by SHession** [comment]
> Is it worth making the preview link column one of the core columns that cannot be hidden?

Yeah that's a good idea, I'll see if I can find out if that would be better

**Comment by paperboyo** [comment]
> Is it worth making the preview link column one of the core columns that cannot be hidden?

Or hijack prefs for Open in… for it? Got some shitty mockups *for discussion only* (to also tighten the spacing as `Preview` is too wordy):

mockup
